inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,45 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+- Protein .pdb file: PDB file containing the structure of the protein to simulate;
+- Temperature of the system in K.
+
+Now, the user is prepared to simulate protein dynamics. Please note that the scenario currently supports only PDB files as input. We have conducted extensive testing with [structures generated by the AlphaFold2 model](https://alphafold.ebi.ac.uk) and [structures from the RCSB database](https://www.rcsb.org). Before simulation, files undergo pre-processing to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.
+
+Also, right now the only supported simulator is `GROMACS`. Further developments will add new simulators to this scenario.
```

Comment:
Remove the second sentence (it is a commitment which we do not know how it will go). Simplify the first to start with "This scenario only supports the GROMACs simulator..."